Soccar [Physics]/src/nl/soccar/physics/GameEngine.java

```diff
@@ -65,13 +65,11 @@ public void run() {
     }
 
     public void stop() {
-        if (game.getStatus() == GameStatus.STOPPED) {
-            return;
-        }
-
         game.stop();
 
         timer.cancel();
+        timer.purge();
+        timer = null;
     }
 
     public void addListener(GameEventListener listener) {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)